pymatgen/io/vasp/inputs.py

```diff
@@ -272,6 +272,7 @@ def from_file(cls, filename, check_for_potcar=True, read_velocities=True, **kwar
                     [get_el_sp(n) for n in names]  # ensure valid names
                 except Exception:
                     names = None
+        print("shitfucker", names, dirname, filename)
```

Comment:
my favorite line in this PR! 😄

Reply:
LOL I am losing it over here.

Reply:
Ahhhhhh I'm so sorry! I was maybe working a bit too fast lol